### DISCUSSION TITLE
Add ParseDMARC?

### DISCUSSION THREAD

**Discussion by AlexanderWright** [docker-mailserver/docker-mailserver#4077] [Ideas]
As a useful addition to the existing DMARC signing already included, can [ParseDMARC](https://domainaware.github.io/parsedmarc/)  be included in the container? 

I think it could be a useful addition.

**Comment by polarathene** [comment]
You could raise a feature request.

The general process we have is that we encourage sharing guides in our docs for configuration, but unless there is quite a bit of user interest voiced or a maintainer is interested in pushing for the integration, we are usually a bit reluctant to adding new features to maintain.

**Comment by tve** [comment]
I've been wanting something that parses the dmarc reports for me and alerts me if there is a delivery issue. It looks like ParseDMARC could do something like this with a bit of post-processing.
But I'm not sure it needs to be integrated with DMS. It seems sufficient to run it on the host once an hour or day on the files that have accumulated in an inbox or directory?

**Comment by audioscavenger** [comment]
if you show me how to set it up and how to get useful intelligence out of it, i would definitely love adding it in the dms-gui project that I work on
